tests/test_uncertainties.py

```diff
@@ -104,9 +105,6 @@ def test_ufloat_fromstr():
         "3.4(nan)e10": (3.4e10, float("nan")),
         # NaN value:
         "nan+/-3.14e2": (float("nan"), 314),
-        # "Double-floats"
```

Comment:
These floats are too large to work with. Prior to this PR the large float value could be saved into the `AffineScalarFunc` object and displayed, but  any arithmetic with such large values would result in an `OverflowError`.  In the new framework even displaying a `UFloat` the first time requires a calculation, even if it's as simple as `sqrt(x**2)`. So in the new framework the `OverflowError` appears immediately.